src/Parser/Hunter/Survival/Modules/Items/ButchersBoneApron.js

```diff
@@ -78,11 +84,64 @@ class ButchersBoneApron extends Analyzer {
     this.bonusDamage += getDamageBonus(event, MODIFIER_PER_STACK * this._savedStacks);
   }
 
+  get apronStacks() {
+    return this._currentStacks;
+  }
+
+  get percentCappedStacks() {
+    return this.wastedStacks / this.totalStacks;
+  }
+
+  get unusedStacks() {
+    return this.totalStacks - this.usedStacks - this.wastedStacks;
+  }
+  get percentUnusedStacks() {
+    return this.unusedStacks / this.totalStacks;
+  }
+
+  get unusedStacksThreshold() {
```

Comment:
This is one of the few cases where I think it makes sense to use a specific number instead of a percentage since it only ever happens once per encounter and a shorter/longer encounter shouldn't change how many stacks it's worth casting it at (I assume at X number of stacks it's worth casting single target? might also be different depending on carve/butchery?).

Reply:
My idea was that it would be less of an issue (since it would be less impactful) over the course of a longer fight than over a shorter fight, but I'll change it to numbers